### PR TITLE
Refactor plot styling

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-543

#### Link to staging deployment URL 

https://ui-iqgequ4xtejbmef4q4pg7twtdr.scp-staging.biomage.net/

#### Links to any Pull Requests related to this

N.A.

#### Anything else the reviewers should know about the changes here

- Refactored plot components
- Refactored `plotStyling` component to use new styling configuration
- Moved the old `plotStyling` component into a `oldPlotStyling` which is to be removed when the component is worked on.

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [X] Tested locally with Inframock
- [X] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [x] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/components/plots/styling/volcano/DisplayLabels.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import _ from 'lodash';
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Slider, Form, Space,
+} from 'antd';
+
+const VolcanoDimensionsRangeEditor = (props) => {
```
Comment (ivababukova) [inline src/components/plots/styling/volcano/DisplayLabels.jsx]: this doesn't sounds right -- needs to be checked

File: src/components/plots/styling/volcano/MarkersEditor.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { Form } from 'antd';
 import ColorBrowser from '../ColorBrowser';
 
-const MarkersEditor = (props) => {
+const VolcanoMarkersEditor = (props) => {
```
Comment (ivababukova) [inline src/components/plots/styling/volcano/MarkersEditor.jsx]: rename the file to match the component name

File: src/components/plots/styling/volcano/ThresholdsGuidesEditor.jsx [hunk 3]
```diff
@@ -43,7 +43,7 @@ ColorPickerOption.propTypes = {
   text: PropTypes.string.isRequired,
 };
 
-const ThresholdsGuidesEditor = (props) => {
+const VolcanoThresholdsGuidesEditor = (props) => {
```
Comment (ivababukova) [inline src/components/plots/styling/volcano/ThresholdsGuidesEditor.jsx]: same here - rename

File: src/pages/experiments/[experimentId]/plots-and-tables/volcano/index.jsx [hunk 4]
```diff
@@ -80,11 +70,66 @@ const VolcanoPlot = ({ experimentId }) => {
...
+      controls: ['legend'],
+    },
+  ];
+
   useEffect(() => {
     if (!config) return;
     setDataPointStatus();
     const generatedSpec = generateSpec(config, plotData);
     setSpec(generatedSpec);
+    console.log(plotStylingConfig);
```
Comment (ivababukova) [inline src/pages/experiments/[experimentId]/plots-and-tables/volcano/index.jsx]: remove that

Comment (aerlaut) [reply src/pages/experiments/[experimentId]/plots-and-tables/volcano/index.jsx]: removed

File: src/components/plots/styling/volcano/DisplayLabels.jsx (hunk 1 above)
Comment (aerlaut) [reply src/components/plots/styling/volcano/DisplayLabels.jsx]: renamed to `DisplayLabels`

File: src/components/plots/styling/volcano/MarkersEditor.jsx (hunk 2 above)
Comment (aerlaut) [reply src/components/plots/styling/volcano/MarkersEditor.jsx]: renamed to `MarkersEditor`

File: src/components/plots/styling/volcano/ThresholdsGuidesEditor.jsx (hunk 3 above)
Comment (aerlaut) [reply src/components/plots/styling/volcano/ThresholdsGuidesEditor.jsx]: renamed to `ThresholdsGuidesEditor`

**Review by ivababukova — APPROVED** [review]

**Pull request merged**